fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/Client.java

```diff
@@ -358,7 +324,7 @@ private void validate() {
 
     }
 
-    private void validateUpdate() {
+    public void validateUpdate() {
```

Comment:
you might wanna move this method into the `ClientDomainServiceImpl`. it is not used anywhere else.

Reply:
Yes, but It has a call to the client method `validateActivationDate` and this is used in other part inside of `Client`